### DISCUSSION TITLE
Cannot create a policy

### DISCUSSION THREAD

**Discussion by nikp86** [eclipse-edc/Samples#251] [Q&A] [answered]
Hello,

I'm following the guide (transfer->transfer-01-negotiation) and got stuck on the point of creating a policy:
![image](https://github.com/eclipse-edc/Samples/assets/42552237/c664f1a7-56f7-46ce-8f14-7764925795d7)

Thank you in advance for your help!

**Comment by ndr-brt** [comment, marked as answer]
windows is not supported as OS, some resolved using WSL, some others still had issue with it, my suggestion is to run the sample from a posix virtual machine

**Reply by nikp86** [reply]
Hello,
thank you for your assistance, you are right!
I'm able to create the policy when I'm working with ubuntu environment!
![image](https://github.com/eclipse-edc/Samples/assets/42552237/e68a8e81-6e06-492d-b751-8a145b86303f)
